test/functional/services/remote/webdriver.ts

```diff
@@ -56,6 +56,12 @@ const downloadDir = resolve(REPO_ROOT, 'target/functional-tests/downloads');
 const chromiumDownloadPrefs = {
```

Comment:
let's rename it to `chromiumUserPrefs` since it is no longer only about downloading